R/dplyr.R

```diff
@@ -70,7 +70,8 @@ mutate.zoomed_dm <- function(.data, ...) {
   tbl <- get_zoomed_tbl(.data)
   mutated_tbl <- mutate(tbl, ...)
   # all columns that are not touched count as "selected"; names of "selected" are identical to "selected"
-  selected <- set_names(setdiff(names(get_tracked_keys(.data)), names(enquos(..., .named = TRUE))))
+  # in case no keys are tracked, `set_names(NULL)` would throw an error
+  selected <- set_names_or_null(setdiff(names(get_tracked_keys(.data)), names(enquos(..., .named = TRUE))))
```

Comment:
If we need to `enquos()`, please extract and call `mutate(!!!quos)` above.

```suggestion
  selected <- set_names(setdiff(names2(get_tracked_keys(.data)), names(quos)))
```

Why would `get_tracked_keys()` return `NULL` anyway, should it return `character()` in this case?

Reply:
Good point, `get_tracked_keys()` now returns an empty named character, when no keys are tracked